src/plugins/layout.ts

```diff
@@ -241,7 +243,10 @@ export default class Layout implements IPlugin {
     let moves = summary.movedNodes.map(this.processMutation.bind(this, Action.Move));
     let updates = summary.updatedNodes.map(this.processMutation.bind(this, Action.Update));
     let removes = summary.removedNodes.map(this.processMutation.bind(this, Action.Remove));
-    let all = [].concat(inserts, moves, updates, removes);
+    let all: IEventData[] = [].concat(inserts, moves, updates, removes);
+    for (let i = 0; i < all.length; i++) {
```

Comment:
Can you add a comment here that includes your response to my question about "why this PR?" please.

Reply:
Done